From 34ce93e1c50bfb88f92a687d4f82de55584f3f6a Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Wed, 11 Apr 2018 14:20:39 +0300
Subject: [PATCH] Do not look into $HOME when looking for gtk modules

On the host it causes host contamination, on the target it's a
potential security issue. Gtk+3 has already removed this.

Upstream-Status: Inappropriate [gtk2 is in maintenance mode]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 gtk/gtkmodules.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/gtk/gtkmodules.c b/gtk/gtkmodules.c
index 50729b6..e09b583 100644
--- a/gtk/gtkmodules.c
+++ b/gtk/gtkmodules.c
@@ -65,10 +65,6 @@ get_module_path (void)
   if (result)
     return result;
 
-  home_dir = g_get_home_dir();
-  if (home_dir)
-    home_gtk_dir = g_build_filename (home_dir, ".gtk-2.0", NULL);
-
   module_path_env = g_getenv ("GTK_PATH");
   exe_prefix = g_getenv ("GTK_EXE_PREFIX");
 
